main/src/cgeo/geocaching/models/PocketQueryList.java

```diff
@@ -62,7 +67,7 @@ private static void selectFromPocketQueries(final Activity activity, final List<
         final CharSequence[] items = new CharSequence[pocketQueryList.size()];
 
         for (int i = 0; i < pocketQueryList.size(); i++) {
-            items[i] = pocketQueryList.get(i).name;
+            items[i] = pocketQueryList.get(i).name + (pocketQueryList.get(i).isDownloadable()?" *":"");
```

Comment:
changed spaces

BTW: this was my quick solution to visually indicate the downloadable PQs in the list. I simply added an asterisk '*' character at the end. Some Icon or other visual differentiation (font weight, color, ...) would be better I guess. But I'm not sure what is the best here.